packages/@aws-cdk-containers/ecs-service-extensions/package.json

```diff
@@ -36,9 +36,7 @@
     "organization": true
   },
   "license": "Apache-2.0",
-  "cdk-build": {
-    "jest": true
-  },
+  "cdk-build": {},
```

Comment:
If we no longer have/need any config for "cdk-build", can we just remove this entirely? (Same throughout).

Reply:
I believe that this change may have something to do with the build failure in aws-cdk-lib. 

```shell
aws-cdk-lib: An error occurred:  TypeError: Cannot read property 'cloudformation' of undefined
```

https://github.com/aws/aws-cdk/blob/d03b586e1f4ac60df021cc8fb591a2672fa7356e/packages/aws-cdk-lib/scripts/verify-stripped-exp.ts#L33